### DISCUSSION TITLE
Will Malt become available to the artist without programming knowledge?

### DISCUSSION THREAD

**Discussion by metkymetky** [bnpr/Malt#292] [General]
Hello @pragma37 
I am new to the Malt project and from looking into it on the surface it seems to be a great tool. Thank you for your hard work!

Also, from what I understood, it seems like this engine requires some programming knowledge.
I wonder if it is planned to make the engine available to the artists without programming knowledge, to have a clear, artist-oriented interface, or is it required to know how to code to use it to the full potential?

**Comment by panthavma** [comment]
Hello!

Malt has a node interface that is pretty much equivalent to code for most purposes, you can see it in the material examples here: https://github.com/bnpr/Malt/discussions/94

You might still need programming *logic* at times, but specific knowledge is not needed imo.

**Reply by pragma37** [reply]
While the nodes are more technical and less documented than the EEVEE/Cycles ones, they are designed to be used by artists too.

You can read the docs here:
https://malt3d.com/Documentation/Graphs/

And here are some (somewhat outdated) introduction videos:
https://www.youtube.com/playlist?list=PLiN2BGdwwlLqbks8h5MohvH0Xd0Zql_Sg

**Comment by eballeste** [comment]
There was a related project called [BEER](https://blendernpr.org/beer/) that worked on top of MALT and would allow you to do a lot of these things via layers in Blenders UI that was in the works but I think that got stalled due to MALT evolving in the way that it has.

**Comment by metkymetky** [comment]
@panthavma @pragma37 @eballeste 
thank you all for your kind answers.
